### PR TITLE
improve channel view cover

### LINKED ISSUE #1612: Lock the channel cover and hero image so they don't move around
As per UI review (#1584), hero image and channel covers should be locked in place so they always display the same content regardless of the viewport size

### PR DESCRIPTION
**Pull request by kdembler**
### Review after #1660 

Resolves #1612 - makes the channel cover image follow roughly the same policy as featured video cover - keep the aspect ratio of the image and let the page content cover it slightly

### CODE REVIEW

File: atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
+import styled from '@emotion/styled'
+import { fluidRange } from 'polished'
+
+import { Avatar, Button } from '@/shared/components'
+import { breakpoints, colors, spacing, typography } from '@/shared/theme'
+import { Link } from '@reach/router'
+
+export const Container = styled.section`
+  position: relative;
+  @media screen and (min-width: ${breakpoints.small}) {
```
Comment (fcescob) [inline atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts]: It looks to me that margins and gradient % need to be in sync based on what "looks good" this is hard to grasp without context, so maybe let's add a comment here describing that.

File: atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    margin-bottom: -75px;
+  }
+  @media screen and (min-width: ${breakpoints.medium}) {
```
Comment (fcescob) [inline atlas/src/components/FeaturedVideoHeader/FeaturedVideoHeader.style.ts]: Not about code, but a big part of the image gets covered on bigger screens 
image

File: atlas/src/views/ChannelView/ChannelView.style.tsx [hunk 3]
```diff
@@ -1,34 +1,88 @@
...
-  background-repeat: no-repeat;
-  height: 430px;
-  padding: 0 var(--global-horizontal-padding);
+
+export const Header = styled.section`
+  position: relative;
+  margin-bottom: 60px;
+
+  @media screen and (min-width: ${breakpoints.small}) {
+    margin-bottom: -75px;
```
Comment (fcescob) [inline atlas/src/views/ChannelView/ChannelView.style.tsx:21]: Same comments as in the `FeaturedVideoHeader` apply here

**Review by fcescob — COMMENTED** [review]
Some minor fixes needed, otherwise looks good.

**Review by fcescob — APPROVED** [review]
LGTM!

**Review by bedeho — APPROVED** [review]

**Pull request merged**